src/main/webapp/WEB-INF/view/profiles.jsp

```diff
@@ -19,7 +19,7 @@
 
 
 <%
-User user = (User) request.getAttribute("user");
+UserStore userStore = (UserStore) request.getAttribute("users");
```

Comment:
This will work, but in general I'd avoid using the store classes directly from JSP. Instead, I'd try to only use attributes that were added from the servlet. This is mostly a style thing and is a bit up to personal preference, but my thinking is that it's easier to know everywhere that calls a store class if you keep it in the servlet classes. It's not the end of the world though so it's up to you.